processor/manager.go

```diff
@@ -39,7 +41,7 @@ func (proc *LifecycleManager) Start() {
 	}
 
 	proc.HandleT.Setup(proc.BackendConfig, proc.gatewayDB, proc.routerDB, proc.batchRouterDB,
```

Comment:
Note: the number of params is keep growing, we need to find a better way to init this.